src/js/utils/element-utils.ts

```diff
@@ -1,6 +1,14 @@
 import { MediaStateReceiverAttributes } from '../constants.js';
 import type MediaController from '../media-controller.js';
 
+export function namedNodeMapToObject(namedNodeMap: NamedNodeMap) {
```

Comment:
question: (per earlier comment) is this needed if we simply rely on properties instead of attributes?

Reply:
Resolving this per prior comments. 👌